Test/UnitTests/TestAddinDescription.cs

```diff
@@ -125,14 +127,47 @@ public void PropertiesFromAddin ()
 			System.Threading.Thread.CurrentThread.CurrentCulture = oldc;
 		}
 
-		[TestCase ("SimpleApp.SystemInfoExtension", "StringResource", "")]
-		public void LocalizerProperties (string addinId, string expectedType, string expectedAssembly)
+		// Built-in
+		[TestCase ("SimpleApp.SystemInfoExtension,0.1.0", "StringResource")]
+		// In own addin
+		[TestCase ("SimpleApp.CommandExtension,0.1.0", "CommandExtension.CustomLocalizerFactory, CommandExtension, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
+		// In dependent addin
+		[TestCase ("SimpleApp.HelloWorldExtension,0.1.0", "CommandExtension.CustomLocalizerFactory, CommandExtension, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
+		// In imported assembly
+		[TestCase ("MultiAssemblyAddin,0.1.0", "SecondAssembly.CustomLocalizerFactory, SecondAssembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
+		public void LocalizerProperties (string addinId, string expectedType)
 		{
 			Addin ad = AddinManager.Registry.GetAddin (addinId);
 			ExtensionNodeDescription localizer = ad.Description.Localizer;
 
 			Assert.AreEqual (expectedType, localizer.GetAttribute ("type"));
-			Assert.AreEqual (expectedAssembly, localizer.GetAttribute ("assembly"));
+		}
+
+		[Test]
+		public void CheckConditionAssemblyNames()
+		{
+			Addin ad = AddinManager.Registry.GetAddin ("SimpleApp.SystemInfoExtension,0.1.0");
+			var conditions = ad.Description.ConditionTypes;
+
+			Assert.AreEqual (0, conditions.Count);
+			foreach (ConditionTypeDescription cond in conditions) { 
```

Comment:
we are missing any assert in the loop?

Reply:
I think I commented on the original PR, I forgot what I wanted to do in this test :joy:

Reply:
https://github.com/mono/mono-addins/pull/156#discussion_r406755517